packages/flutter_tools/test/commands.shard/hermetic/symbolize_test.dart

```diff
@@ -38,6 +40,39 @@ void main() {
     applyMocksToCommand(command);
   });
 
+  testUsingContext('Regression test for type error in codec', () async {
+    StreamTransformer<String, String> testTransformer(Uint8List buffer) {
+      return StreamTransformer<String, String>.fromHandlers(
+        handleData: (String data, EventSink<String> sink) {
+          sink.add(data);
+        },
+        handleDone: (EventSink<String> sink) {
+          sink.close();
+        },
+        handleError: (dynamic error, StackTrace stackTrace, EventSink<String> sink) {
+          sink.addError(error, stackTrace);
+        }
+      );
+    }
+    final DwarfSymbolizationService symbolizationService = DwarfSymbolizationService(
```

Comment:
Maybe a `DwarfSymbolizationService.test()` with this transformer?

Reply:
Done